vertex_ai/workbench_dataproc/main.tf

```diff
@@ -24,7 +24,7 @@ resource "google_workbench_instance" "default" {
     machine_type = "n1-standard-1"
     vm_image {
       project = "deeplearning-platform-release"
```

Comment:
@m-mayran should "deeplearning-platform-release" (UmN) be instead "cloud-managed-notebooks" (GmN and WBI) instead?

Reply:
Also, I'm wondering if this will also affect the guides for User-managed notebooks? Are they also displaying this code?

Reply:
You're right for the project. Updated. 

This code is using `google_workbench_instance` which only works with workbench instances and not UmN (which is google_notebooks_instance)